### PR TITLE
refactor(imports): remove barrel imports

### PR DESCRIPTION
**Pull request by TalDerei**
references #724

eliminates barrel imports to facilitate better tree-shaking (dead code elimination) by bundlers.

### CODE REVIEW

**Comment by TalDerei** [comment]
https://github.com/penumbra-zone/web/issues/736 still unresolved

File: apps/extension/src/routes/popup/settings/settings-full-viewing-key.tsx [hunk 1]
```diff
@@ -45,7 +48,7 @@ export const SettingsFullViewingKey = () => {
                 label={
                   <p className='font-headline font-semibold text-muted-foreground'>Password</p>
                 }
-                onChange={e => {
+                onChange={(e: { target: { value: SetStateAction<string> } }) => {
```
Comment (grod220) [inline apps/extension/src/routes/popup/settings/settings-full-viewing-key.tsx:51]: Don't think we need this as it's an implicit type. Or do we?

File: apps/minifront/src/components/send/ibc/ibc-form.tsx
Comment (grod220) [inline apps/minifront/src/components/send/ibc/ibc-form.tsx:1]: This feels like it shouldn't be an added file, think it's from my other PR. Think that means you need to rebase against latest `main`. I'd suggest squashing all your commits to one first.

File: packages/perspective/package.json [hunk 2]
```diff
@@ -4,16 +4,39 @@
...
+    "./action-view": "./translators/action-view.ts",
+    "./address-view": "./translators/address-view.ts",
+    "./memo-view": "./translators/memo-view.ts",
+    "./output-view": "./translators/output-view.ts",
+    "./spend-view": "./translators/spend-view.ts",
+    "./transaction-view": "./translators/transaction-view.ts",
+    "./types": "./translators/types.ts",
+    "./get-address-view": "./plan/get-address-view.ts",
+    "./index": "./plan/index.ts",
+    "./view-action-plan": "./plan/view-action-plan.ts"
```
Comment (grod220) [inline packages/perspective/package.json:19]: Think we can keep the wild card no?

Comment (TalDerei) [reply packages/perspective/package.json]: modified imports to keep the wild card

**Pull request merged**